src/scripts/backup.sh

```diff
@@ -279,6 +279,16 @@ then
         # Determine the filename to be used as the blob name
         blob_name="automate_`basename $BACKUP_PATH`"
 
+        # Get a list of the backup IDs to remove
+        # and then build up the command that needs to be run at the end
+        # Get list of the backup ids into a json file
+        cmd="chef-automate backup list --result-json backups.json"
+        executeCmd "$cmd"
+
+        # Read in the IDs and build up the removeBackupCmd
+        ids=`cat backups.json | jq -r '[.result.backups[] | .id] | join(" ")'`
+        removeBackupCmd="chef-automate backup delete --yes ${ids} && rm backups.json"
```

Comment:
If ${ids} is an empty string because of no previous backups, what happens then? Script will fail?

Reply:
Script has been updated so that the deletion of backups is not attempted if there is no IDs
In this case the `backups.json` will be the only file deleted